### PR TITLE
added fakend and created new section Services(SAAS or Commercial)

### LINKED ISSUE #4: Faker as API - fakend.fyi
Hi all, i have created this project [fakend.fyi ](https://fakend.fyi/)a free mock API tool basked on faker. I need your help to validate the product. I hope you guys will share reviews about the product.

### PR DESCRIPTION
**Pull request by sanjaykhanssk**

### CODE REVIEW

File: README.md
Comment (ST-DDT) [inline README.md]:
```suggestion
## Services (SAAS or Commercial)

- [Fakend - Production Ready Mock API](https://fakend.fyi) - Create mock API for dev and product demos and supercharge your dev team.
```

**Review by Shinigami92 — APPROVED** [review]

**Review by sanjaykhanssk — COMMENTED** [review]
changes updated

**Review by Shinigami92 — APPROVED** [review]

**Comment by sanjaykhanssk** [comment]
Hi, what are the next steps? it says its blocked.

image

**Comment by matthewmayer** [comment]
> Hi, what are the next steps? it says its blocked.
> 
> image

you dont need to do anything else. a maintainer will merge it when approved.

**Review by matthewmayer — CHANGES_REQUESTED** [review]
As Faker is MIT licensed, I think it would be appropriate to
1. include a courtesy link back to fakerjs.dev homepage, and 
2. include a link to the MIT license from your homepage

For example

`Fakend includes code and data from [Faker.js](https://fakerjs.dev) under a [MIT license](https://raw.githubusercontent.com/faker-js/faker/next/LICENSE).`

**Comment by sanjaykhanssk** [comment]
Definitely yes, I am working on the new version of the product(which includes the commercial side ). It will be released next quarter(AMJ). With that, I'm linking and mentioning most of the open-source libraries used in my product such as Faker, JSONTree, and a couple of other libraries will be linked.

**Comment by Shinigami92** [comment]
> As Faker is MIT licensed, I think it would be appropriate to
> 
> 1. include a courtesy link back to fakerjs.dev homepage, and
> 2. include a link to the MIT license from your homepage
> 
> For example
> 
> `Fakend includes code and data from [Faker.js](https://fakerjs.dev) under a [MIT license](https://raw.githubusercontent.com/faker-js/faker/next/LICENSE).`

Beside the fact that I like that suggestion, I would be in favor if you wont block the merge with such a suggestion
Adding the link can be done without immediately requirement of adding us to their credits, even when this is very nice if they would do

**Review by matthewmayer — APPROVED** [review]

**Comment by matthewmayer** [comment]
ok i accept the promise of @sanjaykhanssk to add the credit soon as sufficient :)

"The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software."

**Comment by Shinigami92** [comment]
image

:thinking: 

... I have approved, I am one of the codeowners :thinking: ...
strange

however I will force merge it ^^

**Pull request merged**
